src/components/TextField/TextField.tsx

```diff
@@ -20,13 +20,15 @@ export const TextField: React.FC<Props> = ({
   required = false,
   onChange = () => {},
 }) => {
-  // generage a unique id once on component load
   const [id] = useState(() => `${name}-${getRandomDigits()}`);
 
-  // To show errors only if the field was touched (onBlur)
-  const [touched, setToched] = useState(false);
+  const [touched, setTouched] = useState(false);
```

Comment:
```suggestion
  const [isTouched, setIsTouched] = useState(false);
```
correct name for boolean variables
FYI